java/src/main/java/com/logicalclocks/hsfs/TrainingDataset.java

```diff
@@ -157,11 +161,8 @@ public void save(Dataset<Row> dataset) throws FeatureStoreException, IOException
    * @throws IOException
    */
   public void save(Query query, Map<String, String> writeOptions) throws FeatureStoreException, IOException {
-    Dataset<Row> dataset = query.read();
-    trainingDatasetEngine.save(this, dataset, writeOptions);
-    if (statisticsEnabled) {
-      statisticsEngine.computeStatistics(this, dataset);
```

Comment:
why did you remove the statistics computation?

Reply:
No I didn't. `save(Query query... ` calls `save(Dataset<Row>` which has the code to compute the statistics

Reply:
ah yes, sorry